dotcom-rendering/src/components/PrivacySettingsLink.importable.tsx

```diff
@@ -0,0 +1,58 @@
+import { css } from '@emotion/react';
+import { cmp, onConsent } from '@guardian/consent-management-platform';
+import type { Framework } from '@guardian/consent-management-platform/dist/types';
+import { palette } from '@guardian/source-foundations';
+import { useState } from 'react';
+import { useOnce } from '../lib/useOnce';
+
+const footerLink = css`
+	color: inherit;
+	text-decoration: none;
+	padding-bottom: 12px;
+	display: block;
+	line-height: 19px;
+
+	:hover {
+		text-decoration: underline;
+		color: ${palette.brandAlt[400]};
+	}
+`;
+
+type Props = {
+	extraClasses?: string;
+};
+
+export const PrivacySettingsLink = ({ extraClasses }: Props) => {
+	const [framework, setFramework] = useState<Framework>();
+
+	useOnce(() => {
+		void onConsent().then((consentState) => {
+			setFramework(consentState.framework ?? undefined);
+		});
+	}, []);
+
+	if (!framework)
+		return (
+			<span css={footerLink} style={{ color: 'transparent' }}>
+				…
+			</span>
+		);
+
+	return (
+		// eslint-disable-next-line jsx-a11y/anchor-is-valid -- that’s what we currently do
+		<a
+			href="#"
```

Comment:
If this were a `button`, how do we ensure it still looks like a link?

Reply:
https://guardian.github.io/csnx/?path=/docs/source-react-components_buttonlink--docs

Reply:
But if we used Source here it would look different to its siblings… unless we override the styles?

Reply:
Turned out the styles worked right out the box: `ButtonLink` it is!